README.md

```diff
@@ -41,3 +34,23 @@ To get involved, please see [Code of Conduct](./CODE_OF_CONDUCT.md) and [contrib
 This repository forms part of the Eiffel Community. Eiffel is a protocol for technology agnostic machine-to-machine communication in continuous integration and delivery pipelines, aimed at securing scalability, flexibility and traceability. Eiffel is based on the concept of decentralized real time messaging, both to drive the continuous integration and delivery system and to document it.
 
 Visit [Eiffel Community](https://eiffel-community.github.io) to get started and get involved.
+
+# Documentation
+
+1. [**Eiffel-RemRem-publish**](wiki/markdown/index.md)
+      - [**Introduction**](wiki/markdown/index.md#Introduction)
+      - [**Compatibility**](wiki/markdown/index.md#Compatibility)
+      - [**Components**](wiki/markdown/index.md#Components)
+      - [**Installation**](wiki/markdown/index.md#Installation)
+
+3. [**Running RemRem-Publish service**](wiki/markdown/running-remrem-publish.md)
+    - [**Run in Docker**](wiki/markdown/docker.md)
+    - [**Configuration**](wiki/markdown/configuration.md)
+    
+4. [**Usage**](wiki/markdown/usage.md)
+    - [**CLI Usage**](wiki/markdown/cliUsage.md)
+    - [**Service Usage**](wiki/markdown/serviceUsage.md)
+
+5. [**Logging**](wiki/markdown/logging.md)
+
+[![](https://jitpack.io/v/Ericsson/eiffel-remrem-publish.svg)](https://jitpack.io/#Ericsson/eiffel-remrem-publish)
```

Comment:
I noticed we still have old links to Jitpack - they do not automatically redirect to the new GitHub organization name so the link should be updated.